float64 t_exec
float64 x_c 
float64 z_c
float64 a_ellps
float64 b_ellps
float64 alpha

bool use_vel_ff
bool use_acc_ff

bool is_forward
